src/navigation/wallet/components/SendToContact.tsx

```diff
@@ -138,7 +140,7 @@ const SendToContact = () => {
             {contacts.map((item, index) => {
               return (
                 <SendContactRow key={index}>
```

Comment:
Could you remove this container (SendContactRow), even delete it because it is no longer used in the whole app and move the `key={index}` to ContactRow. This is because unnecessary top and bottom padding appear in the view.